benefits/core/migrations/0001_initial.py

```diff
@@ -93,8 +93,10 @@ class Migration(migrations.Migration):
             name="PemData",
             fields=[
                 ("id", models.AutoField(primary_key=True, serialize=False)),
-                ("text", models.TextField()),
                 ("label", models.TextField()),
+                ("text", models.TextField(null=True)),
+                ("remote_url", models.TextField(null=True)),
+                ("remote_loaded", models.BooleanField(default=False)),
```

Comment:
Made the `text` property optional, and added a couple properties to load remote data